curated_transformers/models/hf_wrapper.py

```diff
@@ -38,6 +36,10 @@ def build_hf_transformer_encoder_v1(
         mixed_precision=mixed_precision,
         grad_scaler=PyTorchGradScaler(**grad_scaler_config),
     )
+
+    model.init = init
+    model.attrs["encoder"] = encoder
```

Comment:
Do we want to store this in `models.refs` instead (as it's technically a sub-layer and we use that convention in `transformer_model`)?

Reply:
The PyTorch wrapper doesn't add the wrapped model to the `layers` member. The model is wrapped in a shim and added to `shims`. That probably makes sense, because it's not a Thinc model. But I think using `attrs` is probably not good either, because I think the decode is not reentrant after deserialization. Maybe we should just access the shim directly where needed?

Reply:
Fixed. It's slightly ugly, since we are looking at the internals of the shim, but I don't see any better solution.